k8s/upgrade/src/bin/upgrade-job/opts.rs

```diff
@@ -41,21 +41,29 @@ pub(crate) struct CliArgs {
 
     /// The set values specified by the user for upgrade
     /// (can specify multiple or separate values with commas: key1=val1,key2=val2).
-    #[arg(long)]
+    #[arg(long, default_value = "")]
```

Comment:
Should these be Option<String>

Reply:
This is used as a suffix to a command &str/String literal. The empty String makes it easy to append it at the end of the upgrade command. Option would require unwrap blocks, etc. 

So no real reason for this, other than convenience. I'd like to leave this in for now. I am adding this in in this PR so that the binary is usable without these options.